### PR TITLE
Gulp to change urls

### PR DESCRIPTION
**Pull request by shaneoston72**
added gulp tasks to switch api urls for deployment to device and browser development.

### CODE REVIEW

File: config.xml [hunk 1]
```diff
@@ -1,70 +1,75 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<?xml version='1.0' encoding='utf-8'?>
 <widget id="com.ionicframework.starter" version="0.0.1" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
-  <name>HelloCordova</name>
-  <description>
+    <name>HelloCordova</name>
```
Comment (Managram) [inline config.xml:3]: change app name to Leave.

File: config.xml [hunk 2]
```diff
@@ -1,70 +1,75 @@
...
+    <description>
         An Ionic Framework and Cordova project.
     </description>
```
Comment (Managram) [inline config.xml:6]: include our description

File: config.xml [hunk 3]
```diff
@@ -1,70 +1,75 @@
...
-  <author email="you@example.com" href="http://example.com.com/">
+    <author email="you@example.com" href="http://example.com.com/">
       Your Name Here
```
Comment (Managram) [inline config.xml:8]: add our names as well, potentially the link to the front-end github

**Comment by shaneoston72** [comment]
Wrong bratch comparison

**Pull request closed without merging**